src/stories/components/Bar/StackedBar.story.tsx

```diff
@@ -35,7 +35,7 @@ const colors: SpectrumColor[] = [
 const BarStory: StoryFn<typeof Bar> = (args): ReactElement => {
 	const chartProps = useChartProps({ data: barSeriesData, colors, width: 800, height: 600 });
 	return (
-		<Chart {...chartProps}>
+		<Chart {...chartProps} debug>
```

Comment:
This is probably fine for now, but we need to make sure debug doesn't slip into the main branch.  One tip is to manually `git add` files that you see `git status`.  That way you only ever add files with feature changes, not debug logs/flags